example/rootfs_overlay/etc/hosts

```diff
@@ -0,0 +1,6 @@
+127.0.0.1       localhost
+
+## Uncomment this address if you are running NervesHubWeb locally
+## Change IP address to the IP of your local NervesHubWeb server.
+
+# 172.20.193.162  nerves-hub.org
```

Comment:
It seems like `one_dhcpd` should inject `remotehost.eth0` (or something similar) so that you don't need to do this.

Reply:
NervesNetwork has a resolv.conf thing we could abuse also.

Reply:
We would also have to sync work in `nerves_hub_web` because the `Uploader.File` module stores the server information such as host and port in the firmware record. We should revist this to make it easier, but I think its going to require some thought on preprocessing rootfs_overlay files or letting the client dictate the information about the server and only store the path on `nerves_hub_web`.